ci/Jenkinsfile

```
@@ -79,7 +86,7 @@ pipeline {
                     Machine = machine[0].toUpperCase() + machine.substring(1)
```

Should `machine` be indexed?

```suggestion
                    Machine = machine.toUpperCase() + machine.substring(1)
```

---

No.  This is machine name for the projects that is launching subsequent jobs.

---

Oh wait, this is line 86 in the launched jobs.  GOOD QUESTION.  Let me see why I had a [0] there.

---

Let's remove the false index and test it with three labels to test it.

---

Duh, I see now that is simply to capitalize the first letter and as is used in the specification for the labels to be updated.

---

Done.

---

Thanks for the quick reviews David.  Merge when you are ready.